sycl/doc/UsersManual.md

```diff
@@ -1,63 +1,120 @@
+    * spir64_fpga-unknown-unknown-sycldevice - generate code ahead of time for
+      Intel FPGA;
+    * spir64_gen-unknown-unknown-sycldevice - generate code ahead of time for
+      Intel Processor Graphics;
+    Available in special build configuration:
+    * nvptx64-nvidia-cuda-sycldevice - generate code ahead of time for CUDA
+      target;
+
+## Language options
+
+**`-sycl-std=<value>`** [EXPERIMENTAL]
 
-**`-fsycl-targets=<value>`**
+    SYCL language standard to compile for. Possible values:
+    * 121 - SYCL 1.2.1
+    * 2020 - SYCL 2020
+    It doesn't guarantee specific standard compliance, but some selected
+    compiler features change behavior.
+    It is under development and not recommended to use in production
+    environment.
+    Default value is 2020.
 
-    A comma separated list of triples to specify the device target(s) to
-    generate code for.  This option is only valid when used with `-fsycl`.
+**`-f[no-]sycl-unnamed-lambda`**
 
-### Target toolchain options.
+    Enables/Disables unnamed SYCL lambda kernels support.
+    Disabled by default.
 
-**`-Xsycl-target-backend=<triple> <arg>`**
+## Optimization options
 
-    Pass <arg> to the SYCL based backend identified by <triple>.
+**`-f[no-]sycl-early-optimizations`**
 
-**`-Xsycl-target-backend <arg>`**
+    Enables (or disables) intermediate representation optimization pipeline
+    before translation to SPIR-V. Have effect only if optimizations are turned
+    on by standard compiler options (-O1 or higher).
+    Enabled by default.
 
-    Pass <arg> to the SYCL based target backend.
+**`-f[no-]sycl-dead-args-optimization`**
 
-**`-Xsycl-target-frontend=<triple> <arg>`**
+    Enables (or disables) LLVM IR dead argument elimination pass to remove
+    unused arguments for the kernel functions before translation to SPIR-V.
+    Disabled by default.
 
-    Pass <arg> to the SYCL based target frontend identified by <triple>.
+**`-f[no-]sycl-id-queries-fit-in-int`**
 
-**`-Xsycl-target-frontend <arg>`**
+    Assume/Do not assume that SYCL ID queries fit within MAX_INT. It assumes
+    that these values fit within MAX_INT:
+    * id class get() member function and operator[]
+    * item class get_id() member function and operator[]
+    * nd_item class get_global_id()/get_global_linear_id() member functions
+    Enabled by default.
 
-    Pass <arg> to the SYCL based target frontend.
+## Target toolchain options
 
-**`-Xsycl-target-linker=<triple> <arg>`**
+**`-Xsycl-target-backend=<T> "options"`**
+**`-Xs "options"`**
 
-    Pass <arg> to the SYCL based target linker identified by <triple>.
+    Pass "options" to the backend of target device compiler, specified by
+    triple T. The backend of device compiler generates target machine code from
+    intermediate representation. This option can be used to tune code
+    generation for a specific target. The "options" are used during offline
+    compilation and are also saved in a fat binary for online compilation.
```

Comment:
AFAIR, options are only passed through the fat binary for the generic SPIRV target.
+ @mdtoguchi to confirm.
```suggestion
    generation for a specific target. The "options" are used during offline
    compilation and are also saved in a fat binary for online compilation (for the `spir64-unknown-unknown-sycldevice` generic target).
```

Reply:
Yes, that is correct - only for the generic default target.

Reply:
Rephrased.